source/API_Reference/SMTP_API/getting_started_smtp.md

```diff
@@ -0,0 +1,238 @@
+weight: 960
+layout: page
+navigation:
+  show: true
+---
+
+<iframe src="https://player.vimeo.com/video/190122014" width="700" height="400" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+
+You can also send email with [the UI](https://sendgrid.com/docs/User_Guide/Marketing_Campaigns/getting_started.html) and with [the API]({{root_url}}/API_Reference/api_v3.html).
+
+- [What is SMTP?](#-What-is-smtp)
+- [Sending a test SMTP email with Telnet](#-Sending-a-test-SMTP-email-with-Telnet)
+- [Sending an Email with SMTP](#-Sending-an-Email-using-SMTP)
+   - [Ports](#-Ports)
+   - [Rate Limits](#-Rate-Limits)
+- [Customizing Your Send](#-Customizing-Your-Send)
+   - [Filters](#-Filters)
+   - [Scheduling Your Send](#-Scheduling-Your-Send)
+   - [Substitution Tags](#-Substitution-Tags)
+   - [Section Tags](#-Section-Tags)
+   - [Suppression Groups](#-Suppression-Groups)
+   - [Categories](#-Categories)
+   - [Unique Arguments](#-Unique-Arguments)
+
+{% anchor h2 %}
+What is SMTP?
+{% endanchor %}
+
+[SMTP]({{root_url}}/Glossary/smtp.html), or _simple mail transfer protocol_, is a quick and easy way to send email from one server to another. SendGrid provides a 3rd party SMTP service that allows you to deliver your email via our server instead of your own client or server.
+
+For a deeper dive into what SMTP is, the benefits of sending an email with SMTP, and how SendGrid can help, see the [SMTP Service Crash Course](https://sendgrid.com/blog/smtp-service-crash-course/) on our blog.
+
+{% anchor h2 %}
+Sending a test SMTP email with Telnet
+{% endanchor %}
+
+{% anchor h3 %}
+Before you begin
+{% endanchor %}
+
+- Create a SendGrid API key on the [API Keys page](https://app.sendgrid.com/settings/api_keys).
+- Open your command line, bash, shell, or Terminal functionality (depending on what OS you are using). You'll use this window to input the commands to initiate a telnet connection.
+- Convert your API key to Base64. It is not secure to put your API key into an external webpage for a conversion, so we recommend using a bash conversion. If you are on Mac or Linux, you can use the pre-installed OpenSSL package. Use this cmd to convert your API key using OpenSSL: `echo '<<YOUR_API_KEY>>' | openssl base64`. Save your converted key for a later step.
+
+{% warning %}
+Telnet does not register backspaces correctly - so you have to type your commands correctly (or copy and paste it from here).
+{% endwarning %}
+
+*To send SMTP email using Telnet:*
+
+1. Start your session by typing in the terminal: `TELNET smtp.sendgrid.net 25`. SendGrid accepts unencrypted and TLS connections on ports **25**, **467**, **587**, & **2525**. You can also connect via SSL on port **465**. Many hosting providers and ISPs block port 25 as a default practice. If this is the case, contact your host/ISP to find out which ports are open for outgoing SMTP relay. We recommend port 587 to avoid any rate limiting that your server host may apply.

```

Comment:
If you can, I would put a break after the actual step/action so the user can differentiate between the action and the additional information. It might make the page kind of long though, so use your best judement.